tests/__snapshots__/analyze.test.js.snap

```diff
@@ -4,636 +4,7 @@ exports[`should fail if asserting second order index 1`] = `
 Array [
   Array [
     Object {
-      "id": "9",
-      "name": undefined,
-      "passed": true,
```

Comment:
It should return `passed: false` for `should fail if asserting second order index` test.

After adding `analyzeReport.mockClear()` in `afterEach` it stores `passed: false`.